core/src/main/java/oracle/weblogic/deploy/logging/WLSDeployLoggingConfig.java

```diff
@@ -156,6 +156,7 @@ public WLSDeployLoggingConfig() {
             System.exit(ERROR_EXIT_CODE);
         }
         PlatformLogger logger = WLSDeployLogFactory.getLogger(WLSDEPLOY_LOGGER_NAME);   // make sure that this is the first logger
+        // this doesn't appear to do anything.  do we need it?
         logger.info("The {0} program will write its log to {1}", programName, logFileName);
```

Comment:
Seems like this should be removed.  @rakillen @CarolynRountree please comment.

Reply:
Not sure - the message didn't appear in stdout or the log file, so it seems safe to delete.
Carolyn added this with some other logging changes, so not sure if it serves a purpose.

Reply:
Once Carolyn responds, the rest LGTM.

Reply:
remove it